### PR TITLE
Windows SETUP

### PR DESCRIPTION
**Pull request by loomlike**
### Description
Add windows setup instruction

### Related Issues
#567, #491

Related bugs:
#698 (if you don't see this issue anymore, please close)
#709 This is a known issue: mmlspark lightgbm on Windows [https://github.com/Azure/mmlspark/issues/483](https://github.com/Azure/mmlspark/issues/483). I left a comment about how to reproduce the error.

### Checklist:

- [ ] My code follows the code style of this project, as detailed in our [contribution guidelines](../CONTRIBUTING.md).
- [ ] I have added tests.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by roalexan** [comment]
'conda update anaconda' won't work on Windows DSVM. I need to run 'conda install anaconda'. Also, I think '-n root' is only used for Linux? Maybe we should have two sections like:

Linux
```{shell}
conda update conda -n root
conda update anaconda
```

```{bash}
Windows
conda update conda
conda install anaconda
```

File: SETUP.md [hunk 1]
```diff
@@ -94,27 +97,69 @@ To install the environment:
...
+    set PYSPARK_PYTHON=/anaconda/envs/reco_full/python.exe
+    set PYSPARK_DRIVER_PYTHON=/anaconda/envs/reco_full/python.exe
+    set SPARK_HOME_BACKUP=%SPARK_HOME%
+    set SPARK_HOME=
+    set PYTHONPATH_BACKUP=%PYTHONPATH%
+    set PYTHONPATH=
+
+This will export the variables every time we do `conda activate reco_pyspark`.
+To unset these variables when we deactivate the environment,
+create the file `/anaconda/envs/reco_pyspark/etc/conda/deactivate.d/env_vars.bat` and add:
```
Comment (roalexan) [inline SETUP.md]: need c:\ in front, no?

File: SETUP.md [hunk 2]
```diff
@@ -101,65 +101,63 @@ To install the environment:
...
+> </details>
+>
+> <details><summary><strong><em>Windows</em></strong></summary>
+> 
+> To set these variables every time the environment is activated, we can follow the steps of this [guide](https://conda.io/docs/user-guide/tasks/manage-environments.html#windows).
+> Assuming that we have installed the environment in `c:/anaconda/envs/reco_pyspark`,
+> create the file `c:/anaconda/envs/reco_pyspark/etc/conda/activate.d/env_vars.bat` and add:
+> 
+>     @echo off
+>     set PYSPARK_PYTHON=/anaconda/envs/reco_full/python.exe
```
Comment (roalexan) [inline SETUP.md]: Does these a c:/ in front?

File: SETUP.md (hunk 1 above)
Comment (loomlike) [reply SETUP.md]: oh yeah I forgot to add here.

File: SETUP.md [hunk 3]
```diff
@@ -101,65 +101,63 @@ To install the environment:
...
+>     unset SPARK_HOME_BACKUP
+>     ```
+> 
+> </details>
+>
+> <details><summary><strong><em>Windows</em></strong></summary>
+> 
+> To set these variables every time the environment is activated, we can follow the steps of this [guide](https://conda.io/docs/user-guide/tasks/manage-environments.html#windows).
+> Assuming that we have installed the environment in `c:/anaconda/envs/reco_pyspark`,
+> create the file `c:/anaconda/envs/reco_pyspark/etc/conda/activate.d/env_vars.bat` and add:
```
Comment (roalexan) [inline SETUP.md]: shouldn't that slashes point the other way?

File: SETUP.md (hunk 2 above)
Comment (loomlike) [reply SETUP.md]: yes, will add here too. without c: works if anaconda is installed on the same drive as Windows', but if not, should specify I guess.

File: SETUP.md (hunk 3 above)
Comment (loomlike) [reply SETUP.md]: that is correct for Windows. Will update.

File: SETUP.md [hunk 4]
```diff
@@ -101,65 +101,63 @@ To install the environment:
...
+>     set PYSPARK_PYTHON=/anaconda/envs/reco_full/python.exe
+>     set PYSPARK_DRIVER_PYTHON=/anaconda/envs/reco_full/python.exe
+>     set SPARK_HOME_BACKUP=%SPARK_HOME%
+>     set SPARK_HOME=
+>     set PYTHONPATH_BACKUP=%PYTHONPATH%
+>     set PYTHONPATH=
+> 
+> This will export the variables every time we do `conda activate reco_pyspark`.
+> To unset these variables when we deactivate the environment,
+> create the file `/anaconda/envs/reco_pyspark/etc/conda/deactivate.d/env_vars.bat` and add:
```
Comment (roalexan) [inline SETUP.md]: slashes need to change here too

Comment (loomlike) [reply SETUP.md]: Updated

**Comment by miguelgfierro** [comment]
merging since @roalexan approved

**Pull request merged**